app/assets/javascripts/models/beta/task.js

```diff
@@ -0,0 +1,66 @@
+import httpService from '../../services/httpService';
+import changeCase from 'change-object-case';
+import { updateIfSameId } from '../../services/updateIfSameId';
+
+export const post = (projectId, storyId, task) => {
+  const newTask = {
+    name: task,
+    done: false
+  }
+  return httpService
+    .post(`/projects/${projectId}/stories/${storyId}/tasks`, newTask)
+    .then(({ data }) => changeCase.camelKeys(data, { recursive: true, arrayRecursive: true }))
+    .catch((error) => {
+      console.log(error);
+    });
+};
+
+export const update = (projectId, storyId, taskId, status) => {
+  const taskStatus = changeCase.snakeKeys(status);
+  return httpService
+    .put(`/projects/${projectId}/stories/${storyId}/tasks/${taskId}/`, taskStatus)
+    .then(({ data }) => changeCase.camelKeys(data, { recursive: true, arrayRecursive: true }))
+    .catch((error) => {
+      console.log(error);
+    });
+};
+
+export const destroy = (projectId, storyId, taskId) => {
+  return httpService
+    .delete(`/projects/${projectId}/stories/${storyId}/tasks/${taskId}`)
+    .then(({ data }) => changeCase.camelKeys(data, { recursive: true, arrayRecursive: true }))
```

Comment:
Do we need the response of the destroy? What is the content of this response that we need to camelize its keys?

Reply:
We don't really need, thnks!